lib/views/widgets/home_page.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:sun_flutter_capstone/consts/global_style.dart';
+
+class HomePage extends StatelessWidget {
+  const HomePage({ Key? key }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        body: Stack(
+          children: [
+            Container(
+              height: 250,
+              decoration: const BoxDecoration(
+                  gradient: LinearGradient(
+                    begin: Alignment.topRight,
+                    end: Alignment.bottomLeft,
```

Comment:
suggestion to change end property of LinearGradient from Alignment.bottomLeft to **Alignment.bottomRight** to follow Figma's design

![image](https://user-images.githubusercontent.com/86587091/164132385-afb40243-3dde-4f34-9192-e305cd192247.png)